test/unit/conversations_page_test.dart

```diff
@@ -5,7 +5,7 @@ import 'package:test/test.dart';
 
 void main() {
   group('group', () {
```

Comment:
I just noticed the group is named 'group' - how about the group becomes 'ConversationViewModel' and then the tests in the group become 'add' and 'populateWith'?  The test package will print '<group_name> <test_name>' for each test.

Reply:
Actually I think the test name should always describe what the test is doing, ie the expected behaviour. 

for some examples see https://pub.dev/packages/test

In this case maybe something like '.populateWith() replaces _items' ?